components/OverviewFilters.jsx

```diff
@@ -38,18 +39,7 @@ export default function OverviewFilters() {
       </fieldset>
       <fieldset className="cmp-overview-filters__group">
```

Comment:
The more I look at this `fieldset` the more I think it should be within the Availability component... The `fieldset` and `legend` group together and give information about the radio buttons, and the buttons wouldn't be accessible without them. So my "future proofing" thought is that we shouldn't even provide an opportunity for the `fieldset` and `legend` to be missed when using the Availability component. What are your thoughts?